src/Tasks/Microsoft.Packaging.Tools/docs/trimming.md

```diff
@@ -0,0 +1,148 @@
+#Dependency Trimming
```

Comment:
This should not be brought up.  @dsplaisted as we mentioned before the trimming stuff should be left in standard repo.  Just make sure you are able to support the same sort of hook it requires, which is the same as conflict resolution: items to exclude from copy to output by build and publish and exclude from deps file.